npm/oxc-types/types.d.ts

```diff
@@ -1282,22 +1282,20 @@ export type TSTypeQueryExprName = TSImportType | TSTypeName;
 export interface TSImportType extends Span {
   type: 'TSImportType';
   argument: TSType;
-  options: TSImportAttributes | null;
+  options: ObjectExpression | null;
   qualifier: TSTypeName | null;
   typeArguments: TSTypeParameterInstantiation | null;
   isTypeOf: boolean;
 }
 
-export interface TSImportAttributes extends Span {
-  type: 'TSImportAttributes';
-  attributesKeyword: IdentifierName;
-  elements: Array<TSImportAttribute>;
-}
-
 export interface TSImportAttribute extends Span {
```

Comment:
Can this be omitted from the typegen?

Reply:
Good catch! `estree(no_ts_def)` allows omitting types from d.ts.